common/plugins/org.jboss.tools.common.meta.ui/src/org/jboss/tools/common/meta/ui/attribute/editor/PropertiesStringFieldEditor.java

```diff
@@ -12,18 +12,20 @@
 
 import java.beans.PropertyChangeListener;
 import java.util.StringTokenizer;
-import org.jboss.tools.common.model.ui.*;
-import org.jboss.tools.common.model.ui.attribute.editor.*;
-import org.jboss.tools.common.model.ui.objecteditor.*;
-import org.eclipse.jface.util.Assert;
+
+import org.eclipse.core.runtime.Assert;
 import org.eclipse.jface.util.IPropertyChangeListener;
 import org.eclipse.jface.util.PropertyChangeEvent;
 import org.eclipse.swt.layout.*;
 import org.eclipse.swt.widgets.*;
+
 import org.jboss.tools.common.model.*;
 import org.jboss.tools.common.model.options.PreferenceModelUtilities;
-import org.jboss.tools.common.model.util.AbstractTableHelper;
+import org.jboss.tools.common.model.ui.*;

```

Comment:
Should be good to prevent starred imports while refactoring

Reply:
good point. that's a good thing to do while refactoring.